### PR TITLE
Added ErrorParser

### PR DESCRIPTION
**Pull request by OfficialNoob**

### CODE REVIEW

File: popup.js [hunk 1]
```diff
@@ -584,10 +587,25 @@ function clearTimes() {
   resetStartTimeChosen();
 }
 
+var EN_US = new Map();
+
+EN_US.set(400, 'Server said this request was invalid"')
+    .set(429, 'You have submitted too many sponsor times for this one video, are you sure there are this many?')
+    .set(409, 'This has already been submitted before')
+    .set(502, 'It seems the server is down. Contact the dev to inform them.')
+    .set('Unknown', 'here was an error submitting your sponsor times, please try again later.');
```
Comment (ajayyy) [inline popup.js]: You missed a `T`

File: popup.js [hunk 2]
```diff
@@ -584,10 +587,25 @@ function clearTimes() {
...
+
+function ErrorParser (Lang, statusCode) {
+  if(Lang.has(statusCode)) return Lang.get(statusCode);
+  Unknown = Lang.get('Unknown');
+  Unknown +=  Error code " + statusCode;
```
Comment (ajayyy) [inline popup.js]: Missing a quotation. Make sure to test if it builds before giving me the okay to merge.

**Comment by ajayyy** [comment]
Perfect, language support is a great idea!

File: popup.js (hunk 1 above)
Comment (OfficialNoob) [reply popup.js]: Fixed

File: popup.js (hunk 2 above)
Comment (OfficialNoob) [reply popup.js]: I did test just did not copy changes sorry

File: popup.js [hunk 3]
```diff
@@ -593,13 +593,11 @@ EN_US.set(400, 'Server said this request was invalid"')
     .set(429, 'You have submitted too many sponsor times for this one video, are you sure there are this many?')
     .set(409, 'This has already been submitted before')
     .set(502, 'It seems the server is down. Contact the dev to inform them.')
-    .set('Unknown', 'here was an error submitting your sponsor times, please try again later.');
+    .set('Unknown', 'there was an error submitting your sponsor times, please try again later.');
```
Comment (ajayyy) [inline popup.js]: Capital T please :)

Comment (OfficialNoob) [reply popup.js]: Cant you edit that anyway

**Pull request merged**